patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -666,23 +661,23 @@
      public static void func_150900_l()
      {
          func_179214_a(Blocks.field_150350_a, new ItemAir(Blocks.field_150350_a));
-@@ -972,6 +1573,10 @@
+@@ -972,6 +1568,10 @@
          private final float field_78011_i;
          private final int field_78008_j;
  
 +        //Added by forge for custom Tool materials.
-+        @Deprecated public Item customCraftingMaterial = null; // Remote in 1.8.1
++        @Nullable
 +        private ItemStack repairMaterial = null;
 +
          private ToolMaterial(int p_i1874_3_, int p_i1874_4_, float p_i1874_5_, float p_i1874_6_, int p_i1874_7_)
          {
              this.field_78001_f = p_i1874_3_;
-@@ -1006,9 +1611,36 @@
+@@ -1006,9 +1606,33 @@
              return this.field_78008_j;
          }
  
-+        @Deprecated // Use getRepairItemStack below
-         public Item func_150995_f()
+-        public Item func_150995_f()
```

Comment:
This is in the `ToolMaterial` enum and for some reason I thought it was a method added by Forge that happened to be obfuscated because it matches the name of another method in `Item.java`.
Will fix.